docs/acsengine.md

```diff
@@ -1,156 +1,176 @@
-4. **certificate and access config files** - some orchestrators like Kubernetes require certificate generation, and these generated files and access files like the kube config files are stored along side the model and ARM template files.
-
-As a rule of thumb you should always work with the `apimodel.json` when modifying an existing running deployment.  This ensures that all the same settings and certificates are correctly preserved.  For example, if you want to add a second agent pool, you would edit `apimodel.json` and then run `acs-engine` against that file to generate the new ARM templates. Then during deployment all existing deployments remain untouched, and only the new agent pools resources are created.
-
-# Generating a template
-
-Here is an example of how to generate a new deployment.  This example assumes you are using [examples/kubernetes.json](../examples/kubernetes.json).
-
-1. Before starting ensure you have generated a valid [SSH Public/Private key pair](ssh.md#ssh-key-generation).
-2. edit [examples/kubernetes.json](../examples/kubernetes.json) and fill in the blanks.
-3. run `./acs-engine generate examples/kubernetes.json` to generate the templates in the _output/Kubernetes-UNIQUEID directory.  The UNIQUEID is a hash of your master's FQDN prefix.
-4. now you can use the `azuredeploy.json` and `azuredeploy.parameters.json` for deployment as described in [deployment usage](../README.md#deployment-usage).
-
-# Deploying templates
-
-For deployment see [deployment usage](../README.md#deployment-usage).
+# Microsoft Azure Container Service Engine
+
+The Azure Container Service Engine (`acs-engine`) generates ARM (Azure Resource Manager) templates for Docker enabled clusters on Microsoft Azure with your choice of DCOS, Kubernetes, or Swarm orchestrators. The input to acs-engine is a cluster definition file which describes the desired cluster, including orchestrator, features, and agents. The structure of the input files is very similar to the public API for Azure Container Service.
+
+## Install
+
+Binary downloads for the latest verison of acs-engine are available here:
+
+* [OSX](https://github.com/Azure/acs-engine/releases/download/v0.4.0/acs-engine-v0.4.0-darwin-amd64.tar.gz)
+* [Linux 64bit](https://github.com/Azure/acs-engine/releases/download/v0.4.0/acs-engine-v0.4.0-linux-amd64.tar.gz)
+* [Windows 64bit](https://github.com/Azure/acs-engine/releases/download/v0.4.0/acs-engine-v0.4.0-windows-amd64.zip)
+
+Download `acs-engine` for your operating system. Extract the binary and copy it to your `$PATH`.
+
+If would prefer to build `acs-engine` from source or are you are interested in contributing to `acs-engine` see [building from source](#build-from-source) below.
+
+## Usage
+
+`acs-engine` reads a JSON [cluster definition](../clusterdefinition.md) and generates a number of files that may be submitted to Azure Resource Manager (ARM). The generated files include:
```

Comment:
Fan of term "cluster definition" as core user concept in the product. That it ultimately exploded out to an Azure API model never translated as a helpful concept in using.